web/Views/Shared/_Layout.cshtml

```diff
@@ -26,18 +26,27 @@
                         <li class="nav-item">
                             <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
                         </li>
-                        <li class="nav-item" style="margin-left: auto;">
-                            @{
-                                if (User.Identity?.IsAuthenticated == true)
-                                {
-                                    <a href="/TODO/profile" class="nav-link">@(((BricksAndHeartsUser)User.Identity).GoogleEmail)</a>
-                                }
-                                else
+                    </ul>
+                    
+                    <ul class="navbar-nav ms-auto">
+                        @if (User.Identity?.IsAuthenticated == true)
+                        {
+                            <li class="nav-item" style="margin-left: auto;">
+                                <a href="/TODO/profile" class="nav-link">@(((BricksAndHeartsUser)User.Identity).GoogleEmail)</a>
+                            </li>
+                            <li class="nav-item" style="margin-left: auto;">
+                                @using (Html.BeginForm("Logout", "Login", FormMethod.Post))
                                 {
-                                    <a class="nav-link" asp-area="" asp-controller="Login" asp-action="Google">Log in</a>
+                                    <button type="submit" class="btn btn-link nav-link">Log out</button>

```

Comment:
The final commit should probably start with `style` not `feat` - you could change this with a rebase if you wanted to...

Reply:
Does this apply for all styling/CSS-like changes?

Reply:
Yep!